google/cloud/compute/README.md

```diff
@@ -4,6 +4,21 @@ This directory contains an idiomatic C++ client library for the
 [Compute Engine API][cloud-service-docs], a service that lets you create and run
 virtual machines on Google’s infrastructure.
 
+The Compute Engine API has a large surface and is split into multiple services,
+in different subdirectories, each interacting with a specific Compute Engine
+resource. Additionally, Compute Engine is divided into multiple libraries, one
+per resource, to reduce binary size.
+
+CMake targets follow the naming convention
+`google-cloud-cpp::compute_${subdir}`, such that the "disks" library is
+`google-cloud-cpp::compute_disks` and the "instance_group_managers" library is
+`google-cloud-cpp::compute_instance_group_managers`.
+
+Bazel targets follow the naming convention `compute_${subdir}` such that the
+"disks" library is `@com_google_googleapis_google_cloud_cpp//:compute_disks` and
```

Comment:
We call ourselves `@google_cloud_cpp`, e.g.

https://github.com/googleapis/google-cloud-cpp/blob/9caf55b9ffc765db7eda3eb4a1c6e48adb21caf5/google/cloud/compute/quickstart/WORKSPACE.bazel#L25

... but I would probably omit the repo qualifier. I think it can be assumed.